test/test_card.py

```diff
@@ -47,8 +47,8 @@ def test_on_draw(self):
 
 class ActionCardTestCase(TestCase):
     def test_init(self):
-        card = ActionCard('Special action')
-        self.assertEqual('Special action', card.name)
+        card = ActionCard()
+        self.assertEqual(None, card.name)
```

Comment:
`self.assertIsNone` suits more here.
Not to mention the fact that comparing with `None` object by equality is considered a poor practice.